index.html

```diff
@@ -5,9 +5,33 @@
   <meta http-equiv="X-UA-Compatible" content="IE=edge">
   <meta name="viewport" content="width=device-width, initial-scale=1.0">
   <link rel="stylesheet" href="styles.css">
-  <title>Rhodny's Porfolio</title>
+  <link rel="preconnect" href="https://fonts.googleapis.com">
+  <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
+  <link href="https://fonts.googleapis.com/css2?family=Playfair+Display&family=Poppins:wght@400&display=swap" rel="stylesheet">
+  <title>Rhodny's Porfolio Template</title>
 </head>
 <body>
+  <nav class="top-edge-container">
+    <div class="geometry-image"></div>
+    <div class="ic-menu"></div>
+  </nav>
+  <section class="section1-container">
+    <div class="section1">
+      <h1 class="name dark-blue">Charlie Giorgania</h1>
+      <div class="contents">
+        <p class="intro-p dark-blue">
+          Hello! I am a software developer! I can help you build a product, feature or website. Take a look at my work.
+        </p>
+        <p class="intro-p dark-blue">
+          If you like what you see and have a project you need coded, don't hesitate to contact me.
+        </p>
+        <button class="start-collab bg-dark-blue">Start collaboration</button>
+      </div>
+    </div>
+    <div>
+      <div class="thur-image css-image-bg"></div>
+    </div>
```

Comment:
- [ ] It would be better to use background images to be in the background as a `background-img` and not `img` tags for SEO and semantic purposes.